src/components/activity/ActivityDirectivesTablePanel.svelte

```diff
@@ -147,6 +147,15 @@
       sortable: true,
     },
     start_offset: {
+      comparator: (valueA: string, valueB: string) => {
+        if ($plan) {
+          return (
+            getUnixEpochTimeFromInterval($plan.start_time, valueA) -
+            getUnixEpochTimeFromInterval($plan.start_time, valueB)
+          );
+        }
+        return valueA.localeCompare(valueB);
+      },
       field: 'start_offset',
       filter: 'text',
```

Comment:
I'm wondering if we should include a comparator for the `derived_start_time` column as well?

Reply:
CC @duranb

Reply:
I don't think we need to since the format is pretty fixed. I haven't had an issue with sorting DOY formats with the table before (at least none that I've noticed)

Reply:
Sounds good